src/e2e/java/uk/gov/hmcts/reform/sscs/functional/mya/CorIdamService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.sscs.functional.mya;
+
+import java.util.Base64;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.idam.Authorize;
+import uk.gov.hmcts.reform.sscs.idam.IdamApiClient;
+
+@Service
+@Slf4j
+public class CorIdamService {
```

Comment:
Should just be able to use the normal IdamService from common library instead of this class